clients/admin-ui/src/features/privacy-requests/RequestRow.tsx

```diff
@@ -45,10 +42,7 @@ const useRequestRow = (request: PrivacyRequestEntity) => {
   const [hovered, setHovered] = useState(false);
   const [focused, setFocused] = useState(false);
   const [menuOpen, setMenuOpen] = useState(false);
-  const [denyModalOpen, setDenyModalOpen] = useState(false);
-  const [approveModalOpen, setApproveModalOpen] = useState(false);
-  const [approveRequest, approveRequestResult] = useApproveRequestMutation();
-  const [denyRequest, denyRequestResult] = useDenyRequestMutation();
+  const [denyRequest] = useDenyRequestMutation();
```

Comment:
Is this still being used in this component? It looks like `denyRequest` may not be called. I think this is handled in the `useMutation` hook now.